test/plugin/base_test.rb

```diff
@@ -2258,7 +2381,12 @@ def verify_subfields_from_record(payload_key)
       d.emit(payload_key => payload_value)
       d.run
     end
-    verify_log_entries(1, COMPUTE_PARAMS, destination_key) do |entry|
+    expected_params = COMPUTE_PARAMS.dup
+    if payload_key == DEFAULT_LABELS_KEY
+      payload_value = payload_value.merge(COMPUTE_PARAMS[:labels])
+      expected_params[:labels] = payload_value
```

Comment:
Can you do this directly in `log_entry_subfields_params`?

Reply:
Done